docs/content/token/staking/node-operators.mdx

```diff
@@ -36,11 +36,11 @@ If you operate a node and want to allow other token holders to delegate tokens t
 you can [enable delegation for your node](custody-staking-and-delivery#delegation), which allows any user to commit stake for your node 
 and receive a proportional percentage of your node's rewards, as determined by you.
 
-## Unbonding / Withdrawing Stake and Rewards
+## Unstaking / Withdrawing Stake and Rewards
 
-Stake can be [unbonded / withdrawn](custody-staking-and-delivery.mdx#unbonding-/-withdrawing-stake)
+Stake can be [unstaked / withdrawn](custody-staking-and-delivery#unstaking-withdrawing-stake)
 after a week-long lock-up.
 
-Rewards are deposited into the same pool of unlocked tokens where unbonded tokens are stored.
+Rewards are deposited into the same pool of unstaked tokens where unstaked tokens are stored.
```

Comment:
```suggestion
Rewards are deposited into their own pool of reward tokens.
```

Reply:
This whole page needs to be edited, but I saw this so I figured I'd change it now.